plugins/inputs/ping/ping.go

```diff
@@ -171,17 +171,23 @@ func (p *Ping) args(url string) []string {
 	// Build the ping command args based on toml config
 	args := []string{"-c", strconv.Itoa(p.Count), "-n", "-s", "16"}
 	if p.PingInterval > 0 {
-		args = append(args, "-i", strconv.FormatFloat(p.PingInterval, 'f', 1, 64))
+		switch runtime.GOARCH {
+		case "mipsle":
+			// -i argument not supported in busybox
```

Comment:
Let's not add anything for this then, if the ping_interval is set to "0s" it will not add this option.